docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -7,6 +7,11 @@ This is a list of things to do if you want to update from a previous version val
 - take a look at your custom files to see if there are some changes (e.g. `localConfig.js`, `proxy.properties`)
 - Follow the instructions, in order
 
+
+## Migration from 2020.01.01 to 2020.02.00
+
+- eslint-plugin-import has been added in package.json and .eslintrc has been updated with a new rule that check that named imports are valid
+
```

Comment:
```suggestion
```
I don't think this is needed? Do you have some problems after update and npm install?
Maybe on dependend project.
If so, maybe we can add simply add the  .eslintrc to the list of files to check as well as package.json etc...